### PR TITLE
Support `Content-Disposition: inline`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Fixes https://github.com/Homebrew/homebrew-core/pull/32454.

### CODE REVIEW

**Comment by fxcoudert** [comment]
I'm not HTTP-savvy enough to have an opinion here, sorry 😐

File: Library/Homebrew/download_strategy.rb [hunk 1]
```diff
@@ -352,8 +352,9 @@ def resolve_url_and_basename(url)
       end
     end
 
-    filenames = lines.map { |line| line[/^Content\-Disposition:\s*attachment;\s*filename=(["']?)([^;]+)\1/i, 2] }
-                     .compact
+    filenames =
+      lines.map { |line| line[/^Content\-Disposition:\s*(?:inline|attachment);\s*filename=(["']?)([^;]+)\1/i, 2] }
```
Comment (MikeMcQuaid) [inline Library/Homebrew/download_strategy.rb]: Mind using a `do` here and splitting the regex handling onto multiple lines?

**Comment by fxcoudert** [comment]
Possibly `afsctool` too: `https://docs.google.com/uc?export=download&id=0BwQlnXqL939ZQjBQNEhRQUo0aUk`

File: Library/Homebrew/download_strategy.rb (hunk 1 above)
Comment (reitermarkus) [reply Library/Homebrew/download_strategy.rb]: Multi-line chaining looks ugly with `do … end`.

Comment (MikeMcQuaid) [reply Library/Homebrew/download_strategy.rb]: @reitermarkus Cool, fine with me 👍

**Pull request merged**